rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -490,6 +491,9 @@ def _decode_transaction(
                 # for legacy transfers.
                 monerium_special_handling_event = True
 
+            if idx + 1 % 100 == 0:
```

Comment:
make this into a constant and define it somewhere in the decoder constants explaining what it does.